dev/com.ibm.websphere.security.wim.base/resources/com/ibm/websphere/security/wim/util/resources/WebsphereWimUtilMessages.nlsprops

```diff
@@ -18,6 +18,15 @@
 
 # Message prefix block: Profile Manager: 1000-1999
 
+
+DUPLICATE_PROPERTY_EXTENDED=CWIML1996W: The extended property ''{0}'' has already been defined for the ''{1}'' entity and will be ignored.
+DUPLICATE_PROPERTY_EXTENDED.explanation=The extended property has already been defined and will be ignored as there cannot be two properties with the same name.
+DUPLICATE_PROPERTY_EXTENDED.useraction=Check that there are not multiple definitions for the extended property in the server configuration and delete or rename duplicates.
+
```

Comment:
**Just a couple of suggestions for clarity:**

DUPLICATE_PROPERTY_EXTENDED.explanation=The extended property is already defined, and the duplicate definition is ignored, because two properties cannot have the same name.

+DUPLICATE_PROPERTY_EXTENDED.useraction=Check for multiple definitions for the extended property in the server configuration. Delete or rename duplicates.

Reply:
I can do these. I will wait for response on the comment for the message first.